www/i18n/en.json

```diff
@@ -146,6 +146,41 @@
         "no-travel-hint": "To see more, change the filters above or go record some travel!"
     },
 
+    "multilabel":{
+        "walk": "Walk",
+        "e-bike": "E-bike",
+        "bike": "Regular Bike",
+        "bikeshare": "Bikeshare",
+        "scootershare": "Scooter share",
+        "drove_alone": "Gas Car Drove Alone",
+        "shared_ride": "Gas Car Shared Ride",
+        "e_car_drove_alone": "E-Car Drove Alone",
+        "e_car_shared_ride": "E-Car Shared Ride",
+        "moped": "Moped",
+        "taxi": "Taxi/Uber/Lyft",
+        "bus": "Bus",
+        "train": "Train",
+        "free_shuttle": "Free Shuttle",
+        "air": "Air",
+        "not_a_trip": "Not a trip",
+        "no_travel": "No travel",
+        "home": "Home",
+        "work": "To Work",
+        "at_work": "At Work",
+        "school": "School",
+        "transit_transfer": "Transit transfer",
+        "shopping": "Shopping",
+        "meal": "Meal",
+        "pick_drop_person": "Pick-up/ Drop off Person",
```

Comment:
While we are rewriting and have this opportunity, can we make all the text with slashes look `"like / this"` instead of `"like/ this"`?
I believe they were made like that a while ago to force a line break. But we don't do that anymore in the UI so the slashes end up looking weird.

Reply:
I changed the slashes to have spaces before / after in `i18n/en.json` and `json/label-options.json.sample` https://github.com/e-mission/e-mission-phone/pull/1065/commits/36b56daad2be26a6ce0ed1241797f6048707a65e